docker/docker-compose.yml

```diff
@@ -5,30 +5,33 @@ services:
     container_name: "${DOCKER_CONTAINER_API}"
     #restart: always
     depends_on:
-      - mysql
+      - mongodb
     networks:
       - cc
     volumes:
       - "${PROJECT_DIR}:/go/src/app"
-    #environment:
-      #VIRTUAL_HOST: "${DOCKER_CONTAINER_API}"
-  mysql:
-    image: mysql
+    environment:
+      VIRTUAL_HOST: "${DOCKER_CONTAINER_API}"
+  mongodb:
+    image: mongo:3.6.3
     container_name: "${DOCKER_CONTAINER_DB}"
     #restart: always
-    environment:
-      MYSQL_DATABASE: "${DB_NAME}"
-      MYSQL_ROOT_PASSWORD: "${DB_PASSWORD}"
+    volumes:
+      - "./data/db:/data/db"
+    command: mongod --smallfiles --logpath=/dev/null
+    networks:
+      - cc
+  nginx-proxy:
+    image: "${DOCKER_IMAGE_PROXY}"
+    container_name: "${DOCKER_CONTAINER_PROXY}"
+    ports:
+      - 80:80
+      - 443:443
+    volumes:
+      - /var/run/docker.sock:/tmp/docker.sock:ro
+      - .development-ssl-certs:/etc/nginx/certs
     networks:
       - cc
-  #nginx-proxy:
-    #image: jwilder/nginx-proxy
-    #ports:
-      #- "80:80"
-    #volumes:
-      #- /var/run/docker.sock:/tmp/docker.sock:ro
-    #networks:
-      #- cc
 networks:
```

Comment:
These kind of networks creates a new one, doesn't it?

If the project [Micltlán](https://github.com/Cultura-Colectiva-Tech/Mictlan) handles the network, I think we have to use the network of that project instead of creating a new one.

Reply:
Yes, as we talk on Friday we will work with a predefined network but may be we need to talk about it before to solve this by the best way, are you agree?